Tools/clinic/clinic.py

```diff
@@ -4984,25 +5001,22 @@ def run_clinic(parser: argparse.ArgumentParser, ns: argparse.Namespace) -> None:
             parser.error(
                 "can't specify --converters and a filename at the same time"
             )
-        converters: list[tuple[str, str]] = []
-        return_converters: list[tuple[str, str]] = []
-        ignored = set("""
-            add_c_converter
-            add_c_return_converter
-            add_default_legacy_c_converter
-            add_legacy_c_converter
-            """.strip().split())
-        module = globals()
-        for name in module:
-            for suffix, ids in (
-                ("_return_converter", return_converters),
-                ("_converter", converters),
-            ):
-                if name in ignored:
-                    continue
-                if name.endswith(suffix):
-                    ids.append((name, name.removesuffix(suffix)))
-                    break
+        converter_list: list[tuple[str, str, Any]] = []
+        return_converter_list: list[tuple[str, str, Any]] = []
```

Comment:
We have type aliases for converters and return converters; there should be no need to use `Any`.

Reply:
Alternatively, you could just use comprehensions?

Reply:
I failed to please mypy: later, the two separated lists are consumed by the same loop which makes mypy really grumpy.

Reply:
I fixed it in 52da03c2a6d65b22c996527b03e0cccdd01b75ee

Reply:
Hm, I only fixed `ConverterType`. I failed to please mypy with `ReturnConverterType` :(

Reply:
> I fixed it in https://github.com/python/cpython/commit/52da03c2a6d65b22c996527b03e0cccdd01b75ee

That was my first attempt, but mypy tests fail with this :-(

The correct fix is to use `ConverterType | ReturnConverterType`.